.github/workflows/push-images.yaml

```diff
@@ -20,7 +20,7 @@ env:
 
 jobs:
   docker-image-publish:
-    runs-on: [self-hosted, Linux, medium, ephemeral]
+    runs-on: [self-hosted, Linux, large, ephemeral]
```

Comment:
There is currently no large linux runner for hedera sourcify. Still investigating adding cgroups to the check for docker-push-release

Reply:
Thanks for pointing this out. I understand now why the workflow wasn't starting.

Reply:
Thanks @rbarkerSL for enabling `large` self-hosted runner in the repo.